sdk/dotnet/Pulumi/Serialization/Attributes.cs

```diff
@@ -3,7 +3,7 @@
 using System;
 using Google.Protobuf.WellKnownTypes;
 
-namespace Pulumi.Serialization
+namespace Pulumi
 {
     /// <summary>
     /// Attribute used by a Pulumi Cloud Provider Package to mark Resource output properties.
```

Comment:
we should probably update these docs now.  It would actually be good to doc the Stack case here.